data/sql/updates/pending_db_auth/rev_1661693714823101367.sql

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
@Meltie2013  can you please delete this line?

Reply:
Seems to be a git issue when it was pushed.